packages/common-components/src/Button/Button.tsx

```diff
@@ -318,6 +320,7 @@ const Button: React.FC<IButtonProps> = ({
       )}
       disabled={disabled || loading}
       {...rest}
+      data-testid={`${testId}-button`}
```

Comment:
Super nit, I think for consistency we should have 
```suggestion
      data-testid={`button-${testId}`}
```

Reply:
Ok 👍. Agree it's good to keep consistency